src/main/java/cn/nukkit/command/data/CommandEnum.java

```diff
@@ -1,31 +1,21 @@
 package cn.nukkit.command.data;
 
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
 import lombok.ToString;
 
 import java.util.List;
 
 /**
+ * @author lukeeey
  * @author CreeperFace
  */
-@ToString
+@Data
```

Comment:
use only getter/setters, Data is not needed here

Reply:
Well actually `Data` will only generate Getters, ToString, and RequiredArgsConstructor.

It won't generate setters because i declared the fields final and it (shouldnt) generate the hashcode as i already added that.